submissions/cynthiaNwakaeme/JavaScript/script.js

```diff
@@ -0,0 +1,42 @@
+// TASK 1
+var productList = [
+    {
+        productName: "Orgainic African Soap",
+        imageUrl: "../HTML/img/African-Black-Soap.jpg",
+        productPrice:"#1,700",
+    },
+    {
+        productName: "Lucious Black Soap",
+        imageUrl: "../HTML/img/black-soap2.jpg",
+        productPrice: "#2,050",
+    },
+    {
+        productName:"Silk African Soap" ,
+        imageUrl: "../HTML/img/black-soap3.jpg",
+        productPrice:"#659",
+    }
+];
+
+// Using the For Loop
+for (let product = 0; product < productList.length; product++) {
+    console.log(productList[product].productName + " with " + productList[product].imageUrl + " is sold for " + productList[product].productPrice + ".");
+};
+
+// Using the While Loop
+var product = 0;
+while (product < productList.length){
+    console.log(productList[product].productName + " with " + productList[product].imageUrl + " is sold for " + productList[product].productPrice + ".");
+    product++;
+};
+
+// Using the Do ... While Loop
+var product = 0;
```

Comment:
You don't need to declare `product = 0` again since you have done that in line 26

Reply:
Thank You ma. I commented out the code for other loops